Gemfile

```diff
@@ -41,7 +41,7 @@ group :test, :development do
   gem 'factory_girl_rails'
   gem 'rspec-rails'
   gem 'pry-rails'
-  gem 'capybara-webkit'
+  #gem 'capybara-webkit'
```

Comment:
Jamie,

No, I believe it was a problem compiling in my development environment.
Please ignore this.

Dave

On Mon, Feb 23, 2015 at 11:58 AM, Jamie Folsom <notifications@github.com>
wrote:

> In Gemfile
> <https://github.com/hyperstudio/Annotation-Studio/pull/128#discussion_r25180264>
> :
>
> > @@ -41,7 +41,7 @@ group :test, :development do
> >    gem 'factory_girl_rails'
> >    gem 'rspec-rails'
> >    gem 'pry-rails'
> > -  gem 'capybara-webkit'
> > +  #gem 'capybara-webkit'
>
> Was there a specific conflict with capybara-webkit?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hyperstudio/Annotation-Studio/pull/128/files#r25180264>
> .
>